streaming_jupyter_integrations/magics.py

```diff
@@ -474,6 +475,59 @@ def __create_configuration_from_dict(new_values: Dict[str, Any]) -> Configuratio
 
         return configuration
 
+    @line_magic
+    def flink_show_table_tree(self, line: str) -> Tree:
+        return self._build_schema_tree()
+
+    def _build_schema_tree(self) -> Tree:
+        tree = Tree()
+        catalogs = self.st_env.execute_sql("SHOW CATALOGS").collect()
+        for catalog in catalogs:
+            tree.add_node(self._build_catalog_node(catalog))
+        return tree
+
+    def _build_catalog_node(self, catalog) -> Node:
```

Comment:
Please add explicit types to method signatures. I'm sure that `mypy` in `pre-commit` is mentioning it already, but I want to remind about it too.